runtime/rasdump/javadump.cpp

```diff
@@ -3494,17 +3494,16 @@ JavaCoreDumpWriter::writeSegments(J9MemorySegmentList* list, BOOLEAN isCodeCache
 			/* The JIT code cache grows from both ends of the segment: the warmAlloc pointer upwards from the start of the segment
 			 * and the coldAlloc pointer downwards from the end of the segment. The free space in a JIT code cache segment is the 
 			 * space between the warmAlloc and coldAlloc pointers. See jit/codert/MultiCodeCache.hpp, the contract with the JVM is
-			 * that the address of the TR_MCCCodeCache structure is stored at the beginning of the segment, and that the warmAlloc
-			 * and coldAlloc pointers are the first two fields in TR_MCCCodeCache.
+			 * that the address of the TR_MCCCodeCache structure is stored at the beginning of the segment.
 			 */
 			mccCodeCache = *((UDATA**)segment->heapBase);
 			if (mccCodeCache) {
-				warmAlloc = (UDATA *)(*mccCodeCache);
-				coldAlloc = (UDATA *)(*(++mccCodeCache));
+				warmAlloc = (UDATA)_VirtualMachine->jitConfig->codeCacheWarmAlloc(mccCodeCache);
```

Comment:
The jitconfig doesn't exist if we compile without jit support.  This needs to be wrapped in ifdefs
```
J9VM_INTERP_NATIVE_SUPPORT
```
and to check for a null jitConfig

Reply:
@DanHeidinga I have updated the code following your suggestions.